examples/systemd/goodjob.service

```diff
@@ -0,0 +1,92 @@
+# This is an example systemd service configuration that keeps GoodJob running.
+#
+# Customize this file based on your bundler location, app directory, etc.
+#
+# TO RUN AS A USER SERVICE...
+# Customize and copy this file to ~/.config/systemd/user/goodjob.service
+# Then run:
+#   - systemctl --user enable goodjob
+#   - systemctl --user {start,stop,restart,status} goodjob
+# Also you might want to run:
+#   - loginctl enable-linger username
+# So that the service is not killed when the user logs out.
+#
+# TO RUN AS A SYSTEM SERVICE...
+# Customize and copy this to:
+#   (on CentOS) /usr/lib/systemd/system/goodjob.service
+#   (on Ubuntu) /lib/systemd/system/goodjob.service
+# Then run (you may need to use `sudo`):
+#   - systemctl enable goodjob
+#   - systemctl {start,stop,restart,status} goodjob
+#
+# This file corresponds to a single GoodJob process. Add multiple copies
+# to run multiple processes (goodjob-1, goodjob-2, etc).
+#
+# Use `journalctl --unit goodjob -rn 100` to view the last 100 log lines.
+# Or  `journalctl --unit goodjob --follow` to view live log output.
+#
+[Unit]
+Description=GoodJob Background Job Processor
+# Start only once the network is available.
+# If running Postgres locally and it's also managed by systemd, consider adding
+# `postgresql.service` (this list is space-separated).
+After=network.target
+
+# See these pages for lots of options:
+#
+#   https://www.freedesktop.org/software/systemd/man/systemd.service.html
+#   https://www.freedesktop.org/software/systemd/man/systemd.exec.html
+#
+# THOSE PAGES ARE CRITICAL FOR ANY LINUX DEVOPS WORK; read them multiple
+# times! systemd is a critical tool for all developers to know and understand.
+#
+[Service]
+# Type=notify is supported as of GoodJob v3.17.0. In earlier versions, use
+# Type=simple and remove the WatchdogSec line.
+Type=notify
+# If systemd doesn't get pinged by GoodJob at least this often, restart GoodJob.
+WatchdogSec=5s
```

Comment:
Not sure if you have a preferred example interval here. I just threw in 5 seconds.

Reply:
Actually, it just occurred to me to look at the liveness probe example in the readme, which uses 10 seconds. Should I change it to match that?

Reply:
10 seconds was also arbitrary. I don't think it necessitates updating to be consistent; it's an example value.